chatterLink-frontend/src/Components/Chats/Contacts.jsx

```
@@ -46,11 +104,21 @@ function Contacts() {
-              username={user}
-             /> 
-          ))
+          contactList?.length > 0 ? (
+            contactList.map((contact, index) => (
+              <ContactComp
+                index={index}
+                onContactClick={handleContactClick}
+                username={contact.username}
+                userId={contact.userId}
+                selectedIndex={selectedIndex}
+              />))
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)